lib/cublas/util.jl

```diff
@@ -30,9 +30,9 @@ function bandex(A::AbstractMatrix,kl,ku)
     return B
 end
 
-const CublasFloat = Union{Float64,Float32,ComplexF64,ComplexF32}
-const CublasReal = Union{Float64,Float32}
-const CublasComplex = Union{ComplexF64,ComplexF32}
+const CublasFloat = Union{Float64,Float32,ComplexF64,ComplexF32,Float16,ComplexF16}
+const CublasReal = Union{Float16,Float64,Float32}
```

Comment:
Is this safe? Won't this lead to dispatching to CUBLAS wrappers, potentially wrongfully so when the API doesn't have F16 support and we should have used a generic Julia fallback?

Reply:
Yeah, I can remove this and add fallbacks. It's going to lead to a lot of code but I guess that's the CUDA lifestyle for you.